airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -610,7 +610,15 @@ def delete_objects(self, bucket, keys):
         if isinstance(keys, str):
             keys = [keys]
 
-        delete_dict = {"Objects": [{"Key": k} for k in keys]}
-        response = self.get_conn().delete_objects(Bucket=bucket, Delete=delete_dict)
-
-        return response
+        s3 = self.get_conn()
+        batch = 1000
```

Comment:
Can you create a constant and add a comment to explain the reason for this change?

Reply:
I've added a comment and used the chunks method liked you asked. But I'm not going to make a constant out of the batch size, as it's an implementation detail specific only to the `S3Hook.delete_objects` method and will not be re-used elsewhere.